include/psen_scan_v2/scanner_state_machine.hpp

```diff
@@ -81,6 +81,16 @@ inline void ScannerProtocolDef::sendStartRequest(const T& event)
   args_->control_client_->write(StartRequest(args_->config_, DEFAULT_SEQ_NUMBER).serialize());
 }
 
+template <class T>
+inline void ScannerProtocolDef::handleStartRequestTimeout(const T& event)
+{
+  PSENSCAN_DEBUG("StateMachine", "Action: handleStartRequestTimeout");
+  PSENSCAN_ERROR("StateMachine",
+                 "Timeout while waiting for the scanner to start! Retrying... "
+                 "(Please check the ethernet connection or contact PILZ support if the error persists.)");
```

Comment:
If this was missing in the StateMachine refactoring and if it is still important, it is probably best for the future to add a test which checks for this messages.

Reply:
Done.